app/controllers/invites_controller.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class InvitesController < ApplicationController
+  # before_action :set_invite, except: %i[accept]
+
+  def notificate; end
+
+  def accept
+    if invite.nil?
+      redirect_to root_path
+    else
+      @organization_id = invite.organization_id
```

Comment:
Don't use instance variables for something not passed to a view and could be really stored in a local variable.

Reply:
Here need for search organization

Reply:
> Don't use instance variables for something not passed to a view and could be really stored in a local variable.

Here need for search organization

Reply:
> Don't use instance variables for something not passed to a view and could be really stored in a local variable.

Here need for search organization

Reply:
Probably this will be a little better
```suggestion
before_action: :set_invite, except: :index

def accept
  organization = @invite.organization
  Membership.create(organization: organization, user: @invite.user)
  @invite.destroy
  redirect_to organization
end

...

private

def set_invite
  @invite = Invite.find(params[:id])
end
```